Devised by Richard West <r.west@neu.edu> in an attempt to fix the way paths
are manipulated when making anaconda packages. See
https://github.com/conda/conda-recipes/pull/310#issuecomment-106533773

diff --git a/openbabel-2.3.2/src/dlhandler_unix.cpp b/openbabel-2.3.2/src/dlhandler_unix.cpp
index 5bffac3..bc66235 100644
--- src/dlhandler_unix.cpp
+++ src/dlhandler_unix.cpp

@@ -79,7 +79,19 @@ int DLHandler::findFiles (std::vector <std::string>& file_list,
   char buffer[BUFF_SIZE];
 
   if (!path.empty())
-    paths.push_back(path);
+    {
+      strncpy(buffer, path.c_str(), BUFF_SIZE - 1);
+      // add a trailing NULL just in case
+      buffer[BUFF_SIZE - 1] = '\0';    
+      OpenBabel::tokenize(vs, buffer, "\r\n:");
+      if (!vs.empty())
+        {
+          for (unsigned int i = 0; i < vs.size(); ++i) {
+            paths.push_back(vs[i]);
+          }
+        }
+    }
+  
 
   if (getenv("BABEL_LIBDIR") != NULL)
     {
